src/getViewForSwagger2.ts

```diff
@@ -76,6 +77,65 @@ export function getViewForSwagger2(opts: CodeGenOptions): ViewData {
   };
 }
 
+function normalizeResponseDefinitions(swagger: any): any {
+  // ensure that the optional swagger.responses and swagger.definitions fields are present
+  swagger.responses = swagger.responses || {};
```

Comment:
It might be nicer to use a local variable here? We're creating this optional empty object on the swagger object and then later `delete` it.

In any case we're mutating the parameter which isn't really necessary.

Reply:
I'm ok to merge this as is for now.